.bumpversion.cfg

```diff
@@ -0,0 +1,9 @@
+[bumpversion]
+current_version = 1.5.0
+commit = True
+tag = True
+tag_name = v{new_version}
+
+[bumpversion:file:qutebrowser/__init__.py]
+parse = __version__ = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)
+
```

Comment:
nitpick: Don't think there should be a blank line here.

Reply:
Unfortunately, PyCharm (my IDE of choice) maniacally adds newline characters at the end of files. I'll remove this one.

Reply:
I think I managed to remove this finally.